src/components/Camera/StandardCamera.js

```diff
@@ -224,8 +251,43 @@ const StandardCamera = ( ): Node => {
         </View>
       </View>
       <Snackbar visible={showAlert} onDismiss={() => setShowAlert( false )}>
-        {t( "You-can-only-upload-20-media" )}
+        {t( "You-can-only-upload-20-media " )}
       </Snackbar>
+      <StandardBottomSheet snapPoints={["20%"]} hide={discardState.hide}>
```

Comment:
Can we add a backdrop with  `SharedComponents/BottomSheetStandardBackdrop`? A user should be able to tap the backdrop to cancel their photo discard.